container-host-files/etc/hcf/config/scripts/configure-HA-hosts.sh

```diff
@@ -53,6 +56,14 @@ if test -z "${KUBE_SERVICE_DOMAIN_SUFFIX:-}" ; then
     # bootstrap node.
     export KUBE_SERVICE_DOMAIN_SUFFIX="$(awk '/^search/ { print $2 }' /etc/resolv.conf)"
 fi
+
+export KUBE_COMPONENT_INDEX="${HOSTNAME##*-}"
```

Comment:
iirc we had a deterministic way to transform the random suffix of the hostname to a number (to make sure we don't have collisions, which would be bad for something like the diego cell)

Reply:
Fixed in 5d8015fd7feed0a417d51c75d5643364655060c6